packages/redux-routine/tsconfig.json

```diff
@@ -0,0 +1,9 @@
+{
+	"extends": "../../tsconfig.base.json",
+	"compilerOptions": {
+		"rootDir": "src",
+		"declarationDir": "build-types",
+		"noUnusedParameters": false
+	},
+	"include": [ "src/**/*" ]
+}
```

Comment:
Should there be a `references` key here, following what discussed in https://github.com/WordPress/gutenberg/pull/32855#issuecomment-866142581 ?

Reply:
I don't think so because this package has no references 🤔 @sirreal, do we need an empty references at all?

Reply:
Nope. If we don't depend on any other packages in the monorepo, no need to add references 👍